### ISSUE TITLE
How to calculate backward pass with feeding custom gradients.

### ISSUE THREAD

**Issue by yoshi4241link** [tensorflow/tensorflow#21040]
Currently I am replacing Pythorch's code with tensorflow implementation code.

The reference code has the following line.
#####
XXX.backward(grad_output, retain_graph=True)
YYY = input.grad.detach().sum(1).clone().clamp(min=0)
#####
XXX corresponds to ResNet's N*N*C map before fc layers.
In this code, the directly defined gradient is entered instead of the loss value in the calculation of the backward path.

How can I describe this to implement with tensorflow?
